refinery/core/serializers.py

```diff
@@ -27,12 +26,13 @@ class AnalysisSerializer(serializers.ModelSerializer):
     tool_display_name = serializers.SerializerMethodField()
     time_end = DateTimeWithTimeZone()
     time_start = DateTimeWithTimeZone()
+    workflow__uuid = serializers.SerializerMethodField()
 
     class Meta:
         model = Analysis
         fields = ('data_set_uuid', 'facet_name', 'name', 'owner', 'status',
                   'summary', 'time_start', 'time_end', 'tool_display_name',
-                  'uuid', 'workflow')
+                  'uuid', 'workflow', 'workflow__uuid')
```

Comment:
Either use the workflow__uuid or workflow id field. We tend to use uuids, so check if we can just remove the workflow field (returns id).

Reply:
I think we can remove - I just pushed the change so we'll see if testing backs up the lack of usages I saw